c10/core/TensorImpl.h

```diff
@@ -1595,6 +1609,8 @@ struct C10_API TensorImpl : public c10::intrusive_ptr_target {
   // should pack this into a bitfield.
   TensorTypeId type_id_;
   bool is_contiguous_ = true;
+  bool is_channels_last_contiguous_ = false;
+  bool is_channels_last_ = false;
```

Comment:
Would it be possible to add a comment explaining the difference between `is_channels_last_contiguous_` and `is_channels_last_` here? Based on the rest of the patch, I *think* I know what it means, but I don't think it's obvious!